test/compojure/api/integration_test.clj

```diff
@@ -204,8 +218,7 @@
     (fact "Invalid json in body causes 400 with error message in json"
       (let [[status body] (post* app "/models/user" "{INVALID}")]
         status => 400
-        (:type body) => "json-parse-exception"
-        (:message body) => truthy))))
+        (:errors body) => contains "Unexpected character"))))
```

Comment:
I had typo there, check does not work. Should be:
(contains "Unexpected character")